atmo/jobs/models.py

```diff
@@ -263,7 +263,13 @@ def terminate(self):
         if self.latest_run:
             self.cluster_provisioner.stop(self.latest_run.jobflow_id)
 
+    def run_now(self):
+        from .tasks import run_job
+        run_job.delay(self.pk)
+
     def save(self, *args, **kwargs):
+        # whether the job is being created for the first time
+        run_now = self.pk is None
```

Comment:
Does the comment match the code? Do we always want to run the job when saved? If I create a job to only run on weekends I wouldn't want my job to run if I save it on a Wednesday.